<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductList</class>
 <widget class="QWidget" name="ProductList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>풀필먼트 시스템 - 물품 리스트</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>풀필먼트 시스템 > 재고 관리 > 물품 리스트</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="adminLabel">
       <property name="text">
        <string>관리자: 홍길동</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="productListGroup">
     <property name="title">
      <string>물품 리스트</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="searchLayout">
        <item>
         <widget class="QLineEdit" name="searchLineEdit">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>검색어 입력...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="filterComboBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>필터</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>창고별</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>날짜별</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>상품명</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportButton">
          <property name="text">
           <string>내보내기</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="productTable">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>No.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>입고일</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>바코드</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>상품명</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>유통기한</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>수량</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>위치</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="paginationLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="firstPageButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prevPageButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pageLabel">
          <property name="text">
           <string>1 / 5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextPageButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lastPageButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>